sdks/js/src/types.ts

```diff
@@ -0,0 +1,1810 @@
+import moment from "moment-timezone";
```

Comment:
Not sure I'm a big fan of one big file, it's pretty hard to parse, and maintain. Can we maybe have one file per resource and use a "barrel file" to export them all?